VogtObserver2/Trader.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver2
+{
+    class Trader : IObserver
+    {
+        private string _name;
+
+        private Random _rand = new Random();
+
+        private string[] _buyOrSell = new string[] { " ", "buy", "sell" };
```

Comment:
P.s., private hardcoded strings that'll be reused over and over and never change.. should be constants.

`private const string[] _buyOrSell = new string[] { " ", "buy", "sell" };`

When you mark something as constant, it can't change AND it becomes static automatically. Something that is static does not use memory on EVERY class.

Right now, every time u create a Trader, you have to assign new memory to define this array of strings.

If you have it as a const, its only ever created/assigned once in memory, and ALL instances of Trader will use that 1 spot in memory (rather then creating their own).

Imagine ur app had 250-300 traders running? What about hundreds of thousands? It can start to make a difference as things scale.